_posts/2019-01-16-lecture-02.md

```diff
@@ -0,0 +1,459 @@
+
+* Whenever we have a v-structure $$ X_{I-1}  \to X_i \leftarrow X_{I+1}  $$ then $$X_i$$ or one of its descendants are in $$Z$$
+* No other node along the trail is in $$Z$$
+
+Further, for directed graphs that may have more than one trail between nodes "directed separation" (d-separation) gives a notion of separation between nodes.
+
+#### Definition 3.7 D-separation 
+Let $$X,Y,Z$$ be three sets of nodes in $$\mathcal{G}$$. We say that *$$X, Y$$* are d-separated given $$Z$$ (denoted $$ \text{d-sep}_{\mathcal{G}} (X; Y \vert Z) $$ ) if there is no active trail between any node X $$ \in X $$ and Y $$ \in Y $$ given $$Z$$. 
+
+### 3.3.2 Soundness and Completeness
+As a method d-separation has the following properties (proof in text):
+
+* _Soundness_: $$ X, Y $$ are d-separated given $$ Z \implies X \perp Y \vert Z $$ i.e. independence determined by d-separation is satisfied by the underlying distribution. 
+* _Completeness_:  If $$ X \perp Y \vert Z $$ then they are d-separated, i.e. d-separation detects all possible independencies.
+
+***
+
+# In class notes
+
+Motivation: representing joint distributions over some random variables is computationally expensive, so we need methodologies to represent joint distributions compactly.
+
+## Two types of Graphical Models <d-cite key="Blei:2015:BasicsPGM"></d-cite><d-cite key="Jordan:2003:PGM"></d-cite>
+### Directed Graphs (Bayesian Networks)
+An acyclic graph, $$\mathcal{G}$$, is made up of a set of nodes, $$\mathcal{V}$$, and a set 
+of directed edges, $$\mathcal{E}$$, where edges represent a causality relationship between nodes. Nodes in the graph represent a set of random variables, $$\{x_1,...,x_N\}$$, where there is a one-to-one map between nodes and random variables. 
+
+<img src="{{ '/assets/img/notes/lecture-02/directed-graph.png' | relative_url }}" height ="270" width = "480"/>
+
+The joint probability of the above directed graph can be written as follows:
+
+$$(\{X_1,...,X_8\}) = P(X_1)P(X_2)P(X_3|X_1)P(X_4|X_2)P(X_5|X_2)P(X_6|X_3,X_4)P(X_7|X_6)P(X_8|X_5,X_6)$$
+
+### Undirected Graphs (Markov Random Fields) <d-cite key="Jordan:2003:PGM"></d-cite>
+An undirected graph contains nodes that are connected via non-directional edges.
+
+<img src="{{ '/assets/img/notes/lecture-02/undirected-graph.png' | relative_url }}" height ="270" width = "480" />
+
+The joint probability of the above undirected graph can be written as followed:
+
+$$P(\{X_1,...,X_8\})= \frac{1}{Z}exp\bigg(E(X_1)+E(X_2)+E(X_3,X_1)+E(X_4,X_2)+E(X_5,X_2)+E(X_6,X_3,X_4)+E(X_7,X_6)+E(X_8,X_5,X_6) \bigg)$$
+
+## Notation
+
+* Variable: capitalized english letter, with subscripts to represent dimensions (i, j, k) and superscripts to represent index e.g. $$V_{i, j}^j$$ 
```

Comment:
Could you make the notation names in bold? Like `**Variable:**`? (Same as you did in the `The Dishonest Casino` section below.